bundles/org.openhab.automation.pidcontroller/README.md

```diff
@@ -22,20 +22,21 @@ This module triggers whenever the `input` or the `setpoint` changes or the `loop
-| `kd`               | Decimal | D: [Derivative Gain](#derivative-d-gain-parameter) Parameter                                                                                       | Y        |
-| `kdTimeConstant`   | Decimal | D-T1: [Derivative Gain Time Constant](#derivative-time-constant-d-t1-parameter) in sec.                                                            | Y        |
-| `outputLowerLimit` | Decimal | The output of the PID controller will be max this value                                                                                            | Y        |
-| `outputUpperLimit` | Decimal | The output of the PID controller will be min this value                                                                                            | Y        |
-| `loopTime`         | Decimal | The interval the output value will be updated in milliseconds. Note: the output will also be updated when the input value or the setpoint changes. | Y        |
-
-The purpose of the limit parameters are to keep the output value and the integral value in a reasonable range, if the regulation cannot meet its setpoint.
+| Name                 | Type    | Description                                                                                                                                        | Required |
+|----------------------|---------|----------------------------------------------------------------------------------------------------------------------------------------------------|----------|
+| `input`              | Item    | Name of the input [Item](https://www.openhab.org/docs/configuration/items.html) (e.g. temperature sensor value)                                    | Y        |
+| `setpoint`           | Item    | Name of the setpoint Item (e.g. desired room temperature)                                                                                          | Y        |
+| `kp`                 | Decimal | P: [Proportional Gain](#proportional-p-gain-parameter) Parameter                                                                                   | Y        |
+| `ki`                 | Decimal | I: [Integral Gain](#integral-i-gain-parameter) Parameter                                                                                           | Y        |
+| `kd`                 | Decimal | D: [Derivative Gain](#derivative-d-gain-parameter) Parameter                                                                                       | Y        |
+| `kdTimeConstant`     | Decimal | D-T1: [Derivative Gain Time Constant](#derivative-time-constant-d-t1-parameter) in sec.                                                            | Y        |
+| `integralLowerLimit` | Decimal | The I part of the PID controller will be max this value                                                                                            | Y        |
```

Comment:
What is the benefit of renaming? I guess that `outputLowerLimit` is more clear

Reply:
The parameter does not limit the output anymore, but only the integral part.